database/src/main/java/com/pip/chatbot/repository/food/FoodRepository.java

```diff
@@ -40,7 +40,7 @@
     }
 
     public Optional<Restaurant> getRandomRestaurantForCuisine(Integer cityId, String cuisine) {
```

Comment:
To zostawmy po staremu, a sklejanie oceny przenieśmy do serwisu. Są to już nasze zasady biznesowe czemu akurat to robimy, repozytorium powinno umożliwiać wyłącznie dostęp do danych